test/Intl/IntlReturnedValueTests.js

```diff
@@ -6,59 +6,36 @@
 // Return values from Intl function calls show the function name correctly in debugger.
 
 /////////////////// DateFormat ////////////////////
+
 var options = { ca: "gregory", hour12: true, timeZone:"UTC" };
 var dateFormat1 = new Intl.DateTimeFormat("en-US", options);    /**bp:resume('step_over');locals();**/
 WScript.Echo("");  // Dummy line to get desired debugger logging behavior. Required due to the above bug.
 var date1 = new Date(2000, 1, 1);
 
-// Bug: GitHub Issue#3439: Intl function call has 2 entries in the debugger functionCallsReturn.
-// https://github.com/Microsoft/ChakraCore/issues/3439

```

Comment:
Thanks for cleaning up!

Reply:
Was this just cleanup or was something broken?

Reply:
This was just cleanup, I was trying to understand what all of the tests did. The resolution of that bug was that the function call was supposed to have 2 entries by design, so the comment was irrelevant now.

Reply:
Yes you're correct about this comment/cleanup -- I meant the file in general since other lines changed. Looked like just formatting but I wanted to be sure.